junit5/src/main/java/cz/xtf/junit5/extensions/SkipForCondition.java

```diff
@@ -32,8 +31,10 @@ public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext con
 
 	public static ConditionEvaluationResult resolve(SkipFor skipFor) {
 		Image image = Image.resolve(skipFor.image());
-		Pattern name = Pattern.compile(skipFor.name());
-		if (name.matcher(image.getRepo()).matches()) {
```

Comment:
doesn't this break current uses of `name` with regexp ?

Reply:
Yep, although in the case I've been tackling there wasn't any such case and the regex mlogic was introduced just few days ago. 
But yeah, in general you're right. This should be conditional.

Reply:
We want to be able to use regexp in SkipFor against image name from metadata. It is not either or, we want both features in conjuction.

Now we have this 
```
	@SkipFor(image = "eap", name = "eap-xp1-openjdk8-openshift-rhel7", reason = "/opt/eap/standalone/deployments/activemq-rar.rar is not in xp image")
	@SkipFor(image = "eap", name = "jboss-eap-7-eap-xp1-openjdk8-openshift-rhel7", reason = "/opt/eap/standalone/deployments/activemq-rar.rar is not in xp image")
	@SkipFor(image = "eap", name = "eap-xp1-openjdk11-openshift-rhel8", reason = "/opt/eap/standalone/deployments/activemq-rar.rar is not in xp image")
	@SkipFor(image = "eap", name = "jboss-eap-7-eap-xp1-openjdk11-openshift-rhel8", reason = "/opt/eap/standalone/deployments/activemq-rar.rar is not in xp image")
	@Test
	public void testActiveMqRarVersion() throws IOException {
``` 

With comming xp2 it will double. We want to be able to have something like this for all xp versions

```
	@SkipFor(image = "eap", name = "*eap-xp*", reason = "/opt/eap/standalone/deployments/activemq-rar.rar is not in xp image")
	@Test
	public void testActiveMqRarVersion() throws IOException {
```